src/Components/NavSide/index.scss

```diff
@@ -27,30 +27,79 @@
 
   .menu {
     position: absolute;
-    bottom: $tagline-offset;
+    bottom: 1rem;
     background: none;
-    width: 100%;
     text-align: center;
-    // transform: rotate(-90deg);
 
     .menu-item {
-      @include hover-effect();
-
       color: white;
       margin-bottom: 1.25rem;
-      display: block;
-      margin-left: -.15em;
+      overflow: hidden;
+      text-align: left;
+
+      .menu-item__animation-wrapper {
```

Comment:
This can be a code style discussion later, but do you think we want to use this .my-prefix__some-description style even though we're nesting the class using scss? 

In other words, this will expand into: 
```
.menu-item .menu-item__animation-wrapper
```
so should we either not use the prefix so that it would expand to:
```
.menu-item .animation-wrapper
```

or not bother nesting in the scss so the final css would just be:
```
.menu-item__animation-wrapper
```
?

Reply:
Yeah sorry for kind of mucking up the existing style. I'm a fan of BEM but it might not be worth refactoring everything.

Reply:
Cool, all good. Feel free to do this one however. And I'd be down to setup other projects using BEM. I've heard good arguments for using it and would like to try it.